src/main/scala/io/phdata/streamliner/schemacrawler/SchemaCrawlerImpl.scala

```diff
@@ -79,25 +76,35 @@ object SchemaCrawlerImpl extends FileUtil {
       case "INFO" => Level.INFO
       case "TRACE" => Level.FINER
       case _ => Level.ALL
-
     }
-    applyApplicationLogLevel(level)
+    new LoggingConfig(level)
   }
 
   private def getOptions(jdbc: Jdbc): SchemaCrawlerOptions = {
     setLogLevel()
-    val options = SchemaCrawlerOptionsBuilder
-      .builder()
-      .withSchemaInfoLevel(SchemaInfoLevelBuilder.standard())
-      .includeSchemas(new RegularExpressionInclusionRule(jdbc.schema))
-      .tableTypes(jdbc.tableTypes.asJava)
+
+    val options = SchemaCrawlerOptionsBuilder.newSchemaCrawlerOptions()
+    val limitOptionsBuilder = LimitOptionsBuilder.builder()
+    // latest version of schema crawler requires singular names
+    val tableTypes = jdbc.tableTypes
+      .map(i =>
+        i match {
+          case "tables" => "table"
+          case "views" => "view"
+          case s => s
+      })
+      .asJava
+    // schema crawler now names the objects differently, eg Hive."default"
+    limitOptionsBuilder
+      .includeSchemas(new RegularExpressionInclusionRule(".*\"" + jdbc.schema + "\""))
```

Comment:
You _could_ use string interpolation here `".*\"${jdbc.schema}\""`

Reply:
Or just ".*${jdbc.schema}\"+". SchemaCrawler will quote only SQL/ DDL reserved words like "default" (case-insensitive), not all words.

Reply:
I don't think string interpolation works with escaped double quotes. Good call, if it's not always quoted I should remove them.

```object HelloWorld {
	def main(args: Array[String]): Unit = {
	  val name = "brock"
  	println(s"Hello \"${name}\", World!")
	}
}
````
gives me
```
HelloWorld.scala:4: error: value $ is not a member of String
	println(s"Hello \"${name}\", World!")
                          ^
HelloWorld.scala:4: error: value \ is not a member of String
	println(s"Hello \"${name}\", World!")
```

Reply:
Oh that's interesting